### PR TITLE
Valhalla now accepts robots

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
What it says on the tin.

You will now spawn as a race depending on game prefs, same as normal roles (so you can spawn as vatborn too).
## Why It's Good For The Game
Robros get to do stuff in valhalla
## Changelog
:cl:
add: Robots and vatborn have now been accepted in valhalla
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
Finally, the waiting game is over.

**Comment by isotope436** [comment]
nah bring back robot racism

**Comment by Wirpy** [comment]
speedmerge

**Review by TiviPlus — APPROVED** [review]

File: code/datums/jobs/job/fallen.dm [hunk 1]
```diff
@@ -3,6 +3,24 @@
...
+			if(prefs?.robot_type == "Basic")
+				return /mob/living/carbon/human/species/robot
+			if(prefs?.robot_type == "Hammerhead")
+				return /mob/living/carbon/human/species/robot/alpharii
+			if(prefs?.robot_type == "Chilvaris")
+				return /mob/living/carbon/human/species/robot/charlit
+			if(prefs?.robot_type == "Ratcher")
+				return /mob/living/carbon/human/species/robot/deltad
+			if(prefs?.robot_type == "Sterling")
+				return /mob/living/carbon/human/species/robot/bravada
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/datums/jobs/job/fallen.dm]: done

**Comment by Aroliacue** [comment]
Where tf is my speedmerge!!!

**Pull request merged**